test/e2e/framework/deployment/constants.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployment
+
+import (
+	"time"
+)
+
+const (
```

Comment:
Don't think we need a new file for holding unexported functions, what do you think about just putting these in `wait.go`?

Reply:
Have a point there. I wanted to make it clear that the constants are used throughout the pkg but it does look good with wait.go and no "outsider" should notice it.

Reply:
Yeah, I see the original intent though since it was called in both files

Reply:
and fixed this